frontend/src/components/common/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  id: string;
+  icon: string;
+  onAction: (id: string) => void;
+};
+
+const IconButton: FC<Props> = ({ id, icon, onAction }) => {
+  const handleAction = (): void => {
+    onAction(id);
+  };
+
+  return (
+    <button className={styles.button} onClick={handleAction}>
+      <img src={icon} alt="Action" />
+    </button>
```

Comment:
```suggestion
    <Button props />
```
our button component

Reply:
we should use our common `Button` component for `IconButton` 👍

Reply:
I hadn't seen an answer to my question and will ask straight. Why should we use our common Button component here?

Reply:
because we can do it 👍